packages/core/src/components/hotkeys/hotkeys.tsx

```diff
@@ -52,6 +53,7 @@ export class Hotkeys extends AbstractPureComponent<IHotkeysProps, {}> {
 
         let lastGroup = null as string;
         const elems = [] as JSX.Element[];
+        const rootClasses = classNames("pt-hotkey-column", this.props.className);
```

Comment:
move this assignment after the `for` loop, right before the `return` -- this location separates the loop vars from their usage.

Reply:
Moved it!